python/ray/autoscaler/_private/monitor.py

```diff
@@ -148,18 +144,7 @@ def __init__(
         stop_event: Optional[Event] = None,
         retry_on_failure: bool = True,
     ):
-        if not use_gcs_for_bootstrap():
-            # Initialize the Redis clients.
-            redis_address = address
-            self.redis = ray._private.services.create_redis_client(
-                redis_address, password=redis_password
-            )
-            (ip, port) = address.split(":")
-            # Initialize the gcs stub for getting all node resource usage.
-            gcs_address = get_gcs_address_from_redis(self.redis)
-        else:
-            gcs_address = address
-            redis_address = None
+        gcs_address = address
```

Comment:
Or just use `gcs_address` as input arg.

Reply:
I'd prefer if we're maximally conservative about changes to the init args of Monitor.
It's unfortunate that this happened kind of by accident and wasn't broadcast properly, but 
**the Monitor class should be treated as a public interface**.

Reply:
(let's keep the name "address")